package.json

```diff
@@ -17,8 +24,32 @@
   "author": "Procech",
   "license": "MIT",
   "dependencies": {
+    "@babel/polyfill": "^7.12.1",
+    "bcryptjs": "^2.4.3",
+    "body-parser": "^1.19.0",
     "colors": "^1.4.0",
+    "cors": "^2.8.5",
+    "crossenv": "0.0.2-security",
+    "dotenv": "^8.2.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-prettier": "^8.0.0",
+    "express": "^4.17.1",
+    "fs": "0.0.1-security",
```

Comment:
why did you need to install a built in node module?

Reply:
Am removing them